superset/assets/src/visualizations/leaflet_map.js

```diff
@@ -296,9 +326,21 @@ function leafletmap(slice, payload) {
                 }
             },
             pointToLayer: function (feature, latlng) {
-                var styles = getLayerStyles(feature)
-                styles.radius = MARKER_RADIUS;
-                return L.circleMarker(latlng, styles).on('click', mapItemClick);
+              var styles = getLayerStyles(feature)
+              styles.radius = MARKER_RADIUS;
+              var node;
```

Comment:
move
var styles = getLayerStyles(feature)
              styles.radius = MARKER_RADIUS;
to else